cli/atlas/templates/pkg/svc/zserver_test.go.gotmpl

```diff
@@ -20,7 +20,7 @@ func TestGetVersion(t *testing.T) {
 	}
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			server, err := NewBasicServer()
+			server, err := NewBasicServer({{ if .WithDatabase }}nil{{ end }})
```

Comment:
hmm, this provides a good opportunity to demonstrate how to properly stub a database for a unit test.

Or maybe we should set this up to be an integration test, with the whole `e2e` flag and whatnot?

Reply:
yeah! something kinda like that. I don't have a really clear answer, unfortunately, and I don't think anyone else here in Tacoma has an example of one :(